pupil_src/shared_modules/annotations.py

```diff
@@ -221,7 +225,15 @@ def recent_events(self, events):
         while self.annotation_sub.new_data:
             topic, annotation_datum = self.annotation_sub.recv()
             ts = self.g_pool.get_timestamp()
-            logger.info("{} annotation @ {}".format(annotation_datum["label"], ts))
+            world_index = (
+                events["frame.world"][0]["index"]
+                if "frame.world" in events and len(events["frame.world"]) > 0
+                else None
+            )
+            annotation_desc = self._annotation_description(
+                label=annotation_datum["label"], world_index=world_index
+            )
+            logger.info(annotation_desc)
```

Comment:
The frame index is not really meaningful in Capture. Instead, I suggest we display the age of the notification:
```suggestion
            annotation_desc = self._annotation_description(
                label=, world_index=world_index
            )
            annotation_label = annotation_datum["label"]
            annotation_age = ts - annotation_datum["timestamp"]
            annotation_desc = f"{annotation_label} annotation ({annotation_age:.3f} seconds old)"
            logger.info(annotation_desc)
```